p4include/v1model.p4

```diff
@@ -133,14 +145,17 @@ parser Parser<H, M>(packet_in b,
                     inout standard_metadata_t standard_metadata);
 control VerifyChecksum<H, M>(in H hdr,
                              inout M meta);
+@pipeline
```

Comment:
The annotation is on the type, not on the instance, so this way of doing it won't work.
For example, you could very well have multiple blocks with the same type required in the package, e.g., the same signature for checksum updates and for the main pipeline. 

The way you distinguish them is by the argument of package they bind to. That's why there is a pass which takes a policy about which controls to skip. What you have to do is something similar for the P2D2 architecture: a list of controls to skip.